install/kubernetes/helm/istio/requirements.yaml

```diff
@@ -1,11 +1,12 @@
 dependencies:
   - name: pilot
-    version: 0.3.0
-    condition: global.pilot.enabled
+    version: 0.1.0
+    # condition: global.pilot.enabled
```

Comment:
why comment out conditional?

Reply:
Yes mely bad, there is a `enabled` flag as before. I forgot to add it here.